### PR TITLE
feat: added like post endpoint

### PR DESCRIPTION
**Pull request by izzyjosh**
#### Related Issue (Link to issue ticket)
---

#62 

#### Description
---

Enables users to react to post

#### Context
---

​

#### How Has This Been Tested? Screenshots (if appropriate - Postman, etc)
---

- Pytest
Screenshot_20240905-080552

​

#### Types of changes
---

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
      ​

#### Checklist:
---

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: api/v1/models/post.py [hunk 1]
```diff
@@ -24,3 +24,17 @@ class Post(AbstractBaseModel):
 
     def __str__(self) -> str:
         return self.content or self.image or self.video
+
+
+
+class Like(AbstractBaseModel):
+    __tablename__ = "like"
+
+    user_id: Mapped[UUID4] = mapped_column(ForeignKey("user.id"))
```
Comment (codewitgabi) [inline api/v1/models/post.py:33]: User `id` column may not necessarily be a uuid4 type. Any string is acceptable but it has to be unique.

File: api/v1/routes/post.py
Comment (codewitgabi) [inline api/v1/routes/post.py:1]: Was this file formatted??

File: api/v1/services/post.py
Comment (codewitgabi) [inline api/v1/services/post.py:1]: This file doesn't look formatted

File: api/v1/tests/post/test_post_like.py
Comment (codewitgabi) [inline api/v1/tests/post/test_post_like.py:1]: Format this file.

**Review by codewitgabi — CHANGES_REQUESTED** [review]

**Comment by izzyjosh** [comment]
Completed

**Review by codewitgabi — APPROVED** [review]

**Pull request merged**